draft-ietf-quic-recovery.md

```diff
@@ -688,14 +688,14 @@ The recovery period limits congestion window reduction to once per round trip.
 During recovery, the congestion window remains unchanged irrespective of new
 losses or increases in the ECN-CE counter.
 
-## Ignoring Loss of Undecryptable Packets	
-
-During the handshake, some packet protection keys might not be	
-available when a packet arrives. In particular, Handshake and 0-RTT packets	
-cannot be processed until the Initial packets arrive, and 1-RTT packets	
-cannot be processed until the handshake completes.  Endpoints MAY	
-ignore the loss of Handshake, 0-RTT, and 1-RTT packets that might arrive before	
-the peer has packet protection keys to process those packets.	
+## Ignoring Loss of Undecryptable Packets
+
+During the handshake, some packet protection keys might not be
+available when a packet arrives. In particular, Handshake and 0-RTT packets
+cannot be processed until the Initial packets arrive, and 1-RTT packets
+cannot be processed until the handshake completes.  Endpoints MAY
+ignore the loss of Handshake, 0-RTT, and 1-RTT packets that might arrive before
+the peer has packet protection keys to process those packets.
```

Comment:
Why?

Reply:
This is unrelated content. I will trim this commit from the PR.